<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="2" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-07-08T15:41:40 BRT" name="Suite" finished-at="2023-07-08T15:42:17 BRT" duration-ms="37104">
    <groups>
    </groups>
    <test started-at="2023-07-08T15:41:40 BRT" name="Test-1" finished-at="2023-07-08T15:41:55 BRT" duration-ms="15205">
      <class name="rahulshettyacademy.TestDemo">
        <test-method signature="titleCheck()[pri:0, instance:rahulshettyacademy.TestDemo@1c65121]" started-at="2023-07-08T15:41:40 BRT" name="titleCheck" finished-at="2023-07-08T15:41:54 BRT" duration-ms="13966" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at rahulshettyacademy.TestDemo.titleCheck(TestDemo.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheck -->
      </class> <!-- rahulshettyacademy.TestDemo -->
    </test> <!-- Test-1 -->
    <test started-at="2023-07-08T15:41:40 BRT" name="Test-0" finished-at="2023-07-08T15:42:03 BRT" duration-ms="22560">
      <class name="rahulshettyacademy.TestDemo">
        <test-method signature="titleCheck()[pri:0, instance:rahulshettyacademy.TestDemo@10ad20cb]" started-at="2023-07-08T15:41:40 BRT" name="titleCheck" finished-at="2023-07-08T15:42:01 BRT" duration-ms="21428" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at rahulshettyacademy.TestDemo.titleCheck(TestDemo.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheck -->
      </class> <!-- rahulshettyacademy.TestDemo -->
    </test> <!-- Test-0 -->
    <test started-at="2023-07-08T15:42:04 BRT" name="TestGoogle-1" finished-at="2023-07-08T15:42:15 BRT" duration-ms="11513">
      <class name="rahulshettyacademy.TestDemo2">
        <test-method signature="titleCheckGoogle()[pri:0, instance:rahulshettyacademy.TestDemo2@3ee39da0]" started-at="2023-07-08T15:42:04 BRT" name="titleCheckGoogle" finished-at="2023-07-08T15:42:14 BRT" duration-ms="10401" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheckGoogle -->
      </class> <!-- rahulshettyacademy.TestDemo2 -->
    </test> <!-- TestGoogle-1 -->
    <test started-at="2023-07-08T15:41:57 BRT" name="TestGoogle-0" finished-at="2023-07-08T15:42:17 BRT" duration-ms="20465">
      <class name="rahulshettyacademy.TestDemo2">
        <test-method signature="titleCheckGoogle()[pri:0, instance:rahulshettyacademy.TestDemo2@6ac4944a]" started-at="2023-07-08T15:41:57 BRT" name="titleCheckGoogle" finished-at="2023-07-08T15:42:16 BRT" duration-ms="19378" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- titleCheckGoogle -->
      </class> <!-- rahulshettyacademy.TestDemo2 -->
    </test> <!-- TestGoogle-0 -->
  </suite> <!-- Suite -->
</testng-results>
